GDJS/Runtime/cocos-renderers/runtimegame-cocos-renderer.js

```diff
@@ -49,6 +49,17 @@ gdjs.RuntimeGameCocosRenderer.prototype.setFullScreen = function(enable) {
     console.warn("Fullscreen is not implemented yet.");
 };
 
+/**
+ * Checks if the game is in full screen.
+ */
+gdjs.RuntimeGamePixiRenderer.prototype.isFullScreen = function() {
+    var electron = this.getElectron();
+    if (electron) {
+      return electron.remote.getCurrentWindow().isFullScreen();
+    }
+    return window.screen.height === window.innerHeight;
```

Comment:
window might not be defined in Cocos2D-JS on iOS/Android/native executables, might be a good idea to test for its existence.

Reply:
Tbh I cannot even get a Cocos 2d export to run properly. The closest I got is the platformer example working with no graphics showing up only the sounds working. Is cocos2d even meant to work? Has it been abandonned? Will it be woked on?

Reply:
I think we can consider it mostly abandoned. We should probably remove the export option and then remove the code supporting it. For now, it's also fine to not implement this method and just `return false;` with a comment `// Unsupported`.